### ISSUE TITLE
Labels cut off at mesh boundary when rendering in parallel.

### ISSUE THREAD

**Issue by aowen87** [visit-dav/issues-test#854]
If you create a mesh, pseudocolor of u, and label plot of u from multi_rect2d.silo and zoom into the edge of the mesh you will see that the labels will get cut off at the edge of the mesh when scalable rendering is set to always. If you change it to never the labels will appear. I assume that when the image compositing is being done, there must be some strangeness with the z values outside of the mesh and pseudocolor plot.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Eric Brugger
Original creation: 11/21/2017 01:52 pm
Original update: 01/02/2018 06:25 pm
Ticket number: 2983
